packages/changed-elements-react/src/common/FeedbackButton.scss

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+.button {
+  margin: 0px auto 50px;
+  overflow: hidden;
+  position: relative;
+  border: 0px solid black;
+  border-radius: 50px;
+}
+
+.svg {
+  color: white;
+  fill: #ffffff;
+  position: absolute;
+  left: 18px;
+  top: 28%;
```

Comment:
This is an awful way to center an element in CSS. You can use `align-items: center;` in conjunction with `display: grid;` or `display: flex;` on the containing element.

Reply:
I can't seem to get the svg to align with the text properly without using absolute or relative positioning. Any suggestions ?
<img width="104" alt="image" src="https://github.com/iTwin/changed-elements-react/assets/86487204/16f6fd20-3ac3-4a8e-842c-5317fa1f9886">
![image](https://github.com/iTwin/changed-elements-react/assets/86487204/18dc93b7-aacc-49e4-b425-9e31d35ef5cc)

Reply:
I would like a chance to discuss after scrum :D

Reply:
Here:

```tsx
<Button
  as="a"
  className="changed-elems-feedback-btn"
  styleType="high-visibility"
  href={props.feedbackUrl}
  target="_blank"
  rel="noreferrer"
>
  <div style={{ display: "flex", alignItems: "center" }}>
    <SvgSmileyHappy className="changed-elems-feedback-svg" />
    {IModelApp.localization.getLocalizedString("VersionCompare:versionCompare.versionCompareLeaveFeedback")}
  </div>
</Button>
```